src/helpers.tsx

```diff
@@ -0,0 +1,31 @@
+import { Todo } from './types/Todo';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  filter: string,
+  searchQuery: string,
+) => {
+  return todos.filter(todo => {
```

Comment:
It would be the best if you don't filter the array if no query and filter equals `all`